content/en/about/contributing/syntax/_index.md

```diff
@@ -85,33 +122,33 @@ Will render as:
 
 {{% alert title="Warning" color="warning" %}} This is a warning. {{% /alert %}}
 
-### Including file snippets
+### Code or shell snippets
 
-The code should be surrounded with three backticks and include the file type as
-a parameter.
+The code or instructions should be surrounded with three backticks and include
+the file type as a parameter.
 
 The supported languages are dependant on the syntax highlighter, which depends
 itself on the mardkown parser.
 
 {{% alert title="Note" color="info" %}} For [Hugo](https://gohugo.io/) the
 [goldmark parser](https://github.com/yuin/goldmark) is used and it relies on the
-[Chroma highlighter](https://github.com/alecthomas/chroma). {{% /alert %}}
+[Prism syntax highlighting](https://prismjs.com/download.html#themes=prism). {{% /alert %}}
 
 The list of languages supported by chroma can be found in their
 [GitHub repository](https://github.com/alecthomas/chroma#supported-languages).
```

Comment:
If chroma is no more used this should be replaced/removed.

Reply:
Will replace it in the new PR. For now I restored the current content under this section.